src/lib/fs-db.js

```diff
@@ -84,7 +97,7 @@ class FsDb {
 
   mapFor(filepath, create = true) {
     const getCallback = create ? this.createOrGetMap.bind(this) : this.getMap.bind(this);
-    const keys = this.toRelpath(path.dirname(filepath)).split('/').filter(x => x);
+    const keys = this.toRelpath(path.posix.dirname(filepath)).split('/').filter(x => x);
```

Comment:
Instead of using path.posix so many times, I think you could change the requiere to `requiere: const path = require('path').posix;` and just use path